liberty-managed/src/main/java/io/openliberty/arquillian/managed/WLPManagedContainer.java

```diff
@@ -398,6 +387,9 @@ public ProtocolMetaData deploy(final Archive<?> archive) throws DeploymentExcept
       try {
          // If the deployment is to server.xml, then update server.xml with the application information
          if (containerConfiguration.isDeployTypeXML()) {
+             if (log.isLoggable(FINER)) {
+                 log.finer("Deploying using server.xml");
```

Comment:
This is OK but I might have included the archiveName in the log msg.

Reply:
I don't think having the archive name here is necessary as it'll be included with the archive details (logged at the same level on line 376)